Fix non-unified build for ScrollingTreeOverflowScrollingNodeRemoteMac

pgorszkowski-igalia opened the pull request:
#### 7cc2d2294c4f9bf6353bcfd04c6d75db0f29e52b
<pre>
Fix non-unified build for ScrollingTreeOverflowScrollingNodeRemoteMac
<a href="https://bugs.webkit.org/show_bug.cgi?id=251841">https://bugs.webkit.org/show_bug.cgi?id=251841</a>

Reviewed by Adrian Perez de Castro.

Fix non-unified build for error:
ScrollingTreeOverflowScrollingNodeRemoteMac.cpp:45:15: error: member access into incomplete type &apos;WebCore::ScrollingTreeScrollingNodeDelegate&apos;

* Source/WebKit/UIProcess/RemoteLayerTree/mac/ScrollingTreeOverflowScrollingNodeRemoteMac.cpp:

Canonical link: <a href="https://commits.webkit.org/259956@main">https://commits.webkit.org/259956@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/3e6aafb36b01a96bd05dd81e0ae983c760d4a2fc

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/106522 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/15538 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/39331 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/115707 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/115356 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/17033 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/6766 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/98714 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/112290 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/12940 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/95910 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/35/builds/40501 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/94815 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/27554 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/82220 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/8767 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/28906 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/9310 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/84/builds/5481 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/14926 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/48454 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/6884 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/10848 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

aperezdc reviewed (approved):
@pgorszkowski-igalia Just FYI, typically build fixes can land unreviewed. In that case if you prefix first line of the commit log with `Unreviewed ...` then the merge-queue will be happy to land the patch without needing a `Reviewed by ...` line.

pgorszkowski-igalia committed:
Fix non-unified build for ScrollingTreeOverflowScrollingNodeRemoteMac
https://bugs.webkit.org/show_bug.cgi?id=251841

Reviewed by Adrian Perez de Castro.

Fix non-unified build for error:
ScrollingTreeOverflowScrollingNodeRemoteMac.cpp:45:15: error: member access into incomplete type 'WebCore::ScrollingTreeScrollingNodeDelegate'

* Source/WebKit/UIProcess/RemoteLayerTree/mac/ScrollingTreeOverflowScrollingNodeRemoteMac.cpp:

Canonical link: https://commits.webkit.org/259956@main
